storage/localstore/localstore_test.go

```diff
@@ -600,7 +611,7 @@ func TestDBDebugIndexes(t *testing.T) {
 	}
 
 	// assert that there's a pin and gc exclude entry now
-	testIndexCounts(t, 1, 1, 0, 1, 1, 1, 0, indexCounts)
+	testIndexCounts(t, 1, 1, 0, 1, 1, 1, 1, indexCounts)
```

Comment:
ditto

Reply:
Access index is replaced with meta index which contains store timestamp. Access index did not have an entry, but meta index now has. Test helper testIndexCounts may be better constructed so that it is obvious which indexes are tested. I do not think that this function is designed nicely.